model_analyzer/config/generate/quick_plus_concurrency_sweep_run_config_generator.py

```diff
@@ -153,3 +157,9 @@ def _set_concurrency(self, run_config: RunConfig, concurrency: int) -> RunConfig
             perf_config.update_config({"concurrency-range": concurrency})
 
         return run_config
+
```

Comment:
This method is duplicated (also in brute search). Maybe this should be a static method in ModelProfileSpec?

Reply:
Yes, this is still duplicated. I didn't clean it up yet. Both classes implement ConfigGeneratorInterface. You could create a base class with common code if you want.